s3-proxy/s3-proxy/src/test/java/com/woowacourse/s3proxy/web/infrastructure/S3StorageTest.java

```diff
@@ -0,0 +1,108 @@
+package com.woowacourse.s3proxy.web.infrastructure;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.BDDMockito.given;
+
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.model.PutObjectResult;
+import com.woowacourse.s3proxy.common.FileFactory;
+import com.woowacourse.s3proxy.exception.upload.UploadFailureException;
+import com.woowacourse.s3proxy.web.domain.PickGitStorage.StoreResult;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.test.util.ReflectionTestUtils;
+import org.springframework.web.multipart.MultipartFile;
+
+@ExtendWith(MockitoExtension.class)
+class S3StorageTest {
+
+    @InjectMocks
+    private S3Storage s3Storage;
+
+    @Mock
+    private FileNameGenerator fileNameGenerator;
+
+    @Mock
+    private AmazonS3 amazonS3;
+
+    private String fileUrlFormat = "https://djgd6o993rakk.cloudfront.net/images/%s";
+
+    @BeforeEach
+    void setUp() {
+        ReflectionTestUtils.setField(s3Storage, "fileUrlFormat", fileUrlFormat);
+    }
+
+    @DisplayName("이미지 파일을 업로드한 뒤, 결과를 반환한다.")
+    @Test
+    void store_Results_Success() {
+        List<MultipartFile> multipartFiles = List.of(
+            FileFactory.getTestRightImage1(),
+            FileFactory.getTestRightImage2()
+        );
+        String userName = "kevin";
+
+        given(fileNameGenerator.generate(any(MultipartFile.class), eq(userName)))
+            .willCallRealMethod();
+        given(amazonS3.putObject(any(), any(), any(), any()))
+            .willReturn(new PutObjectResult());
+
+        List<StoreResult> actual = s3Storage.store(multipartFiles, userName);
+        List<StoreResult> expected = generateSuccessResults(multipartFiles, userName);
+
+        assertThat(actual).usingRecursiveComparison()
+            .isEqualTo(expected);
```

Comment:
```suggestion
        assertThat(actual)
            .usingRecursiveComparison()
            .isEqualTo(expected);
```

usingRecursiveComparison() 쓰면 보통 개행 이렇게 하더라구요?!